federator/integrationTest/integrationTest.js

```diff
@@ -39,6 +39,14 @@ const sideKeys = process.argv[3] ? process.argv[3].replace(/ /g, '').split(',')
 const mainchainFederators = getMainchainFederators(mainKeys);
 const sidechainFederators = getSidechainFederators(sideKeys, sideConfig);
 
+const ONE_DAY_IN_SECONDS = 24*3600;
```

Comment:
I think you need to rebase from nftbridge, these integration test changes are not from this PR I think

Reply:
My bad, it was missing the push --force after rebase (: